implants/lib/eldritch/src/runtime/eprint_impl.rs

```diff
@@ -0,0 +1,65 @@
+use crate::runtime::{messages::ReportErrorMessage, Environment};
+use anyhow::Result;
+
+pub fn eprint(env: &Environment, message: String) -> Result<()> {
+    env.send(ReportErrorMessage {
+        id: env.id(),
+        error: message.clone(),
```

Comment:
unnecessary clone, move the `eprintln!` above this and borrow `&message` in that call

Reply:
I cannot.
error expects a String not &String
and we use message later in `eprintln!`